### ISSUE TITLE
Test Datasets: The LDCT's test results are inconsistent with other papers

### ISSUE THREAD

**Issue by Haoran740** [workingcoder/EDCNN#4]
In the experiment where the ratio of training set to test set is 9 to 1, the index of LDCT in your paper is 36.7594, but my personal test result is 40+, which is closer to REDCNN.
I would like to ask if you adjusted the contrast of the picture before conducting the experiment, and if so, what is the value of the contrast?
Thank you

**Comment by ykm4013** [comment]
Hello, could you please open source your training code？

**Comment by ykm4013** [comment]
> In the experiment where the ratio of training set to test set is 9 to 1, the index of LDCT in your paper is 36.7594, but my personal test result is 40+, which is closer to REDCNN. I would like to ask if you adjusted the contrast of the picture before conducting the experiment, and if so, what is the value of the contrast? Thank you

Hello, could you please open source your training code？

**Comment by Haoran740** [comment]
> > In the experiment where the ratio of training set to test set is 9 to 1, the index of LDCT in your paper is 36.7594, but my personal test result is 40+, which is closer to REDCNN. I would like to ask if you adjusted the contrast of the picture before conducting the experiment, and if so, what is the value of the contrast? Thank you
> 
> Hello, could you please open source your training code？

Sorry, I am currently out of my previous research environment and unable to provide you with training code. However, you can refer to the open source code of the UForm model.

**Comment by ykm4013** [comment]
> > > In the experiment where the ratio of training set to test set is 9 to 1, the index of LDCT in your paper is 36.7594, but my personal test result is 40+, which is closer to REDCNN. I would like to ask if you adjusted the contrast of the picture before conducting the experiment, and if so, what is the value of the contrast? Thank you
> > 
> > 
> > Hello, could you please open source your training code？
> 
> Sorry, I am currently out of my previous research environment and unable to provide you with training code. However, you can refer to the open source code of the UForm model.
> 
> > > 在训练集和测试集比例为9比1的实验中，你论文中LDCT的指数是36.7594，但我个人测试结果是40+，更接近REDCNN。我想问一下，在进行实验之前，您是否调整了图片的对比度，如果是，对比度的值是多少？谢谢你
> > 
> > 
> > 你好，请问你能开源你的训练代码吗？
> 
> 对不起，我目前已经脱离了以前的研究环境，无法为您提供培训代码。但是，您可以参考UForm模型的开放源代码。

Thank you so much for your assistance

**Comment by ykm4013** [comment]
> > > In the experiment where the ratio of training set to test set is 9 to 1, the index of LDCT in your paper is 36.7594, but my personal test result is 40+, which is closer to REDCNN. I would like to ask if you adjusted the contrast of the picture before conducting the experiment, and if so, what is the value of the contrast? Thank you
> > 
> > 
> > Hello, could you please open source your training code？
> 
> Sorry, I am currently out of my previous research environment and unable to provide you with training code. However, you can refer to the open source code of the UForm model.

Thank you so much for your assistance.   At your convenience, can you provide specific information about the UForm model . Like which paper it was proposed in, or its specific address on github.

**Comment by hale1a1** [comment]
> In the experiment where the ratio of training set to test set is 9 to 1, the index of LDCT in your paper is 36.7594, but my personal test result is 40+, which is closer to REDCNN. I would like to ask if you adjusted the contrast of the picture before conducting the experiment, and if so, what is the value of the contrast? Thank you

Hello, the dataset link provided in the project has expired. Could you please share your dataset link?

**Comment by hale1a1** [comment]
> In the experiment where the ratio of training set to test set is 9 to 1, the index of LDCT in your paper is 36.7594, but my personal test result is 40+, which is closer to REDCNN. I would like to ask if you adjusted the contrast of the picture before conducting the experiment, and if so, what is the value of the contrast? Thank you

I also encountered this problem. Have you solved it yet?
